compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -175,10 +175,16 @@ writeIfacesAndHie ifDir files =
 
 -- For backwards compatibility we allow a file at the source root level and just take it's directory
 -- to be the source root.
```

Comment:
For completeness... maybe adjust that comment here back to reality. The paragraph about `topdir` in the description after the imports could be moved here.